src/Database/Schema/TableSchema.php

```diff
@@ -834,6 +834,33 @@ public function dropConstraintSql(Connection $connection)
 
         return $dialect->dropConstraintSql($this);
     }
+
+    /**
+     * Returns an array of the table schema.
+     *
+     * @return array
+     */
+    public function toArray()
+    {
+        return [
+            'columns' => $this->_columns,
```

Comment:
Shouldn't this output include the table's name?

Reply:
$this->myTable->getSchema()->toArray()

In this case I know the table from the myTable property, right?

Reply:
In that scenario yes, but schema objects could be used in other contexts and knowing the table name would be useful.